R/utils.R

```diff
@@ -185,12 +185,15 @@ SparseMMFromFactor <- function(thefactor) {
 }
 
 
-## slm.fit.csr with a fix
+## slm.fit.csr with a fix and using lm.fit backend
```

Comment:
I'll suggest "variant of slm.fit.csr", or similar, in case updating the roxygen block gets forgotten when the code gets updated.

Reply:
Changed to reflect this comment.